src/lambdaDestination.ts

```diff
@@ -0,0 +1,17 @@
+import * as lambda from "@aws-cdk/aws-lambda";
+import * as logs from "@aws-cdk/aws-logs";
+import { Construct } from "@aws-cdk/core";
+
+/**
+ * Use a Lamda Function as the destination for a log subscription.
+ * Using this class temporarily until https://github.com/aws/aws-cdk/pull/14222 is merged and released.
+ */
+export class LambdaDestination implements logs.ILogSubscriptionDestination {
```

Comment:
Basically I am locally implementing [this](https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/aws-logs-destinations/lib/lambda.ts#L9) but specifically leaving out the logic to add lambda permissions because they are unnecessary as the forwarder is deployed with [this](https://github.com/DataDog/datadog-serverless-functions/blob/083957182b8475406d443d70391a6f5018102973/aws/logs_monitoring/template.yaml#L554)